pkg/wait/conditions.go

```diff
@@ -102,10 +107,28 @@ func DoesResourceExist(ctx context.Context, kubeClient *client.Client, resource
 	}
 }
 
+// AreNodesReadyWithinRange returns a WaitCondition that checks if the number of ready nodes matching the given labels is within the expected range
+func AreNodesReadyWithinRange(ctx context.Context, kubeClient *client.Client, expectedNodes Range, labels *cr.MatchingLabels) WaitCondition {
```

Comment:
If we have `IsNumNodesReady`, why not `IsNumNodesReadyWithinRange`?
Or rename both to `AreNumOfNodesReady` and `AreNumOfNodesReadyWithinRange`?

Reply:
It just sounded nicer to me with `Are` instead of `Is`. I think it should also be `AreNumNodesReady`, but we are already using it with `Is`. Don't know how other people feel about it

Reply:
I vote for consistency. Don't have strong opinions on which. 
As we're not using it too much yet changing names now is ok as updating the tests would be fairly quick.

Reply:
Let's change it and use `AreNumNodes...` for both then.